### PR TITLE
added logging and setup fossology to only use good ciphers

### PR DESCRIPTION
**Pull request by patrickpierson**

### CODE REVIEW

File: scripts/start_fossology.sh [hunk 1]
```diff
@@ -28,6 +28,8 @@ Listen 443
     SSLEngine on
     SSLCertificateFile "${SERVER_CRT}"
     SSLCertificateKeyFile "${SERVER_KEY}"
+    SSLProtocol -all +TLSv1 +TLSv1.1 +TLSv1.2
```
Comment (matthewkmayer) [inline scripts/start_fossology.sh]: Should we accept anything other than TLS 1.2?  Using 1.1 and 1.2 is probably okay, but I think we could leave out TLS 1.0.

Comment (patrickpierson) [inline scripts/start_fossology.sh]: I was going off of what I thought I saw the return was for the other services that were updated.  Let me double check.

Comment (patrickpierson) [inline scripts/start_fossology.sh]: My mistake.  Updating for just tls 1.2

**Comment by patrickpierson** [comment]
$ nmap --script +ssl-enum-ciphers -p 443 localhost

Starting Nmap 7.40 ( https://nmap.org ) at 2017-08-02 15:36 EDT
Nmap scan report for localhost (127.0.0.1)
Host is up (0.000074s latency).
Other addresses for localhost (not scanned): ::1
PORT    STATE SERVICE
443/tcp open  https
| ssl-enum-ciphers: 
|   TLSv1.2: 
|     ciphers: 
|       TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 (dh 2048) - A
|       TLS_DHE_RSA_WITH_AES_256_CBC_SHA (dh 2048) - A
|       TLS_DHE_RSA_WITH_AES_256_CBC_SHA256 (dh 2048) - A
|       TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 (dh 2048) - A
|       TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384 (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 (secp256r1) - A
|     compressors: 
|       NULL
|     cipher preference: client
|_  least strength: A

Nmap done: 1 IP address (1 host up) scanned in 0.31 seconds

**Pull request merged**